src/fides/connectors/aws.py

```diff
@@ -244,7 +244,7 @@ def create_redshift_systems(
             fides_key=cluster["ClusterIdentifier"],
             name=cluster["ClusterIdentifier"],
             description=f"Fides Generated Description for Redshift Cluster: {cluster['ClusterIdentifier']}",
-            meta={(pair["Key"], pair["Value"]) for pair in cluster.get("Tags", {})},
+            meta={pair["Key"]: pair["Value"] for pair in cluster.get("Tags", {})},
```

Comment:
Was this just something we missed in the original implementation? Returning a set of tuples instead of a dictionary?

Reply:
This was something I updated after a merge with main yesterday, it's brand new, I think it might have been getting coerced with pydantic v1, and not coerced in Pydantic v2.